io/disk/Avago_storage_adapter/avago9361_vd.py

```diff
@@ -150,6 +156,99 @@ def test_online_offline(self):
                 time.sleep(10)
         self.vd_delete()
 
+    def test_ghs_dhs(self):
+
+        """
+        Test to create Global hot spare and Dedicated hot spare
+        """
+        self.vd_create('WT', 'nora', 'direct', 512)
+        cmd = "./storcli64 /c%d/%s add hotsparedrive" % (self.controller,
+                                                         self.hotspare)
+        self.check_pass(cmd, "Failed to create GHS")
+        self.set_online_offline('offline')
+        self.rebuild_progress(self.hotspare)
+        self.copyback_operation()
+        cmd = "./storcli64 /c%d/%s delete hotsparedrive" % (self.controller,
+                                                            self.hotspare)
+        self.check_pass(cmd, "Failed to delete hotsparedrive")
```

Comment:
I don't think code path traversed in both cases is same.
Deleting hotspare and then deleting vd is different from deleting vd directly.
If procedure is former, try to stick with the right procedure.